packages/app-bridge/src/types/Color.ts

```diff
@@ -134,4 +127,33 @@ export type ColorCreate = {
     alpha: number;
 };
 
-export type ColorPatch = Partial<Omit<Color, 'id' | 'hex' | 'hue' | 'saturation' | 'lightness'>>;
+export type ColorPatch = Partial<Omit<ColorRevision, 'hex' | 'hue' | 'saturation' | 'lightness'>> & {
+    name?: Nullable<string>;
+    sort?: number;
+};
+
+type Rgba = {
+    red: Nullable<number>;
+    green: Nullable<number>;
+    blue: Nullable<number>;
+    alpha: Nullable<number>;
+};
+
+type Cmyk = {
+    c: Nullable<number>;
+    m: Nullable<number>;
+    y: Nullable<number>;
+    k: Nullable<number>;
+    cmykCoated: Nullable<string>;
+    cmykUncoated: Nullable<string>;
+    cmykNewspaper: Nullable<string>;
+};
+
+type Pantone = {
+    pantone: Nullable<string>;
```

Comment:
this key becomes redundant as such: I would suggest `code` or `reference` instead

Reply:
`code` seems good